packages/hub/src/components/layout/UserControls.tsx

```diff
@@ -22,28 +24,40 @@ export function UserControls({ session }: { session: Session | null }) {
     // https://github.com/vercel/next.js/issues/42556 (it's closed but not really solved)
     window.location.href = chooseUsernameRoute();
   }
+  const router = useRouter();
 
   return !session?.user ? (
     <Button onClick={() => signIn()}>Sign In</Button>
   ) : (
     <div className="flex items-center gap-2">
-      {session.user.username === undefined ? null : (
-        <div>
-          <UsernameLink username={session.user.username} />
-        </div>
-      )}
-
-      <DotsDropdown>
-        {() => (
+      <Dropdown
+        render={() => (
           <DropdownMenu>
+            <DropdownMenuHeader>User Actions</DropdownMenuHeader>
```

Comment:
`<DropdownMenuSeparator />` after the header?

Reply:
Good call